Src/main.c

```diff
@@ -108,8 +117,27 @@ int main(void)
   /* USER CODE BEGIN WHILE */
   while (1)
   {
+    //Starts the ADC
+    HAL_ADC_Start(&hadc1);
+    HAL_ADC_PollForConversion(&hadc1, HAL_MAX_DELAY);
+
+    // Will range from 0 - 4095
+    adcValue = HAL_ADC_GetValue(&hadc1);
+
+    adcPercentage = adcValue/4059.0;
+
+    //Needs to be a value between 1 and 2ms so the it will be the percentage + 1 (50% is really 1.5ms)
+    //That value needs to then be scalled by the timer period and desired period = 60000/20 = 3000
+    pwmValue = (adcPercentage*(HIGH - LOW) + LOW) * 3000.0;
```

Comment:
In general, I'm not a huge fan of magic numbers. I would probably pull them out of here and define them as `const`s above so that there is some more context.

Reply:
Math should be good though!